### PR TITLE
Add last_sync_task and pulp_labels to e-e/repositories/

### PR DESCRIPTION
**Pull request by rochacbruno**
issue_433
Issue: AAH-433

### CODE REVIEW

File: galaxy_ng/app/api/ui/serializers/execution_environment.py [hunk 1]
```diff
@@ -63,6 +63,10 @@ def get_pulp(self, distro):
                 'name': repo.name,
                 'description': repo.description,
                 'pulp_created': repo.pulp_created,
+                'last_sync_task': _get_last_sync_task(repo),
+                'pulp_labels': {
+                    label.key: label.value for label in repo.pulp_labels.all()
```
Comment (rochacbruno) [inline galaxy_ng/app/api/ui/serializers/execution_environment.py:68]: @newswangerd I was unsure if labels here are from `repository` or `distribution` both have labels.

**Comment by gerrod3** [comment]
Hey, just want to give a heads up that we found that pulp_labels weren't being exposed for AnsibleDistributions, https://pulp.plan.io/issues/8441. We have a simple fix being added for the next version, https://github.com/pulp/pulp_ansible/pull/547. Would you guys need this in a new 0.7.z release?

**Comment by rochacbruno** [comment]
> Hey, just want to give a heads up that we found that pulp_labels weren't being exposed for AnsibleDistributions, [pulp.plan.io/issues/8441](https://pulp.plan.io/issues/8441). We have a simple fix being added for the next version, [pulp/pulp_ansible#547](https://github.com/pulp/pulp_ansible/pull/547). Would you guys need this in a new 0.7.z release?

Hi @gerrod3 in this case here we are fetching it from `pulp_container`, I am not sure if we need it for collections, what do you think @newswangerd ?

File: galaxy_ng/app/api/ui/serializers/execution_environment.py (hunk 1 above)
Comment (newswangerd) [reply galaxy_ng/app/api/ui/serializers/execution_environment.py:68]: @rochacbruno you can leave this on the repository. Lets add it to the distribution as well.

**Review by newswangerd — APPROVED** [review]
just need to add labels on the distribution as well. Looks good.

**Pull request merged**